2025-10-04 06:31:21,684 - JARVIS - INFO - [MainThread] - __init__:82 - J.A.R.V.I.S. system initialized
2025-10-04 06:31:21,792 - JARVIS.MemoryManager - INFO - [MainThread] - initialize:78 - Initializing memory manager...
2025-10-04 06:31:21,792 - JARVIS.MemoryManager - INFO - [MainThread] - _initialize_vector_database:100 - Initializing SentenceTransformer embedding model...
2025-10-04 06:31:45,334 - JARVIS.MemoryManager - INFO - [MainThread] - _initialize_vector_database:102 - Embedding model initialized
2025-10-04 06:31:45,664 - JARVIS.MemoryManager - ERROR - [MainThread] - initialize:92 - Error initializing memory manager: name 'asyncio' is not defined
2025-10-04 06:31:45,665 - JARVIS.Updater - INFO - [MainThread] - initialize:64 - Initializing update manager...
2025-10-04 06:31:45,666 - JARVIS.Updater - INFO - [MainThread] - initialize:76 - Update manager initialized
2025-10-04 06:31:45,666 - JARVIS.Updater - INFO - [MainThread] - check_for_updates:93 - Checking for updates...
2025-10-04 06:31:45,760 - JARVIS.Updater - ERROR - [MainThread] - _fetch_version_info:336 - HTTP error fetching version info: HTTPSConnectionPool(host='api.jarvis.ai', port=443): Max retries exceeded with url: /updates/version.json (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000001F702B6EC10>: Failed to resolve 'api.jarvis.ai' ([Errno 11001] getaddrinfo failed)"))
2025-10-04 06:31:45,874 - JARVIS.Updater - INFO - [MainThread] - check_for_updates:93 - Checking for updates...
2025-10-04 06:31:45,876 - JARVIS.Updater - ERROR - [MainThread] - _fetch_version_info:336 - HTTP error fetching version info: HTTPSConnectionPool(host='api.jarvis.ai', port=443): Max retries exceeded with url: /updates/version.json (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000001F702B6F4D0>: Failed to resolve 'api.jarvis.ai' ([Errno 11001] getaddrinfo failed)"))
